index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul hasan portfolio</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <style> body{background: url("image/pic.png");} </style>
+
+  </head>
+
+  <body>
+    <header>
+    <nav>
+    <div class="navbar">
+            <h1 class="logo"> My Logo </h1>
+
+            <ul>
+              <li class="nav-items"><a href="#">Portfolio</a></li>
+              <li class="nav-items"><a href="#">About</a></li>
+              <li class="nav-items"><a href="#">Contact</a></li>
+            </ul>
```

Comment:
- [ ] [OPTIONAL] These navigations will be required later on in the desktop version. I suggest you keep them out for now.